app/script/components/list/conversationListCell.js

```diff
@@ -24,38 +24,31 @@ window.z.components = z.components || {};
 
 z.components.ConversationListCell = class ConversationListCell {
   constructor({conversation, is_selected = z.util.noop, click = z.util.noop}) {
-    this.on_in_viewport = this.on_in_viewport.bind(this);
-
     this.conversation = conversation;
     this.is_selected = is_selected;
     this.on_click = click;
 
-    this.entered_viewport = ko.observable(false);
     this.users = ko.pureComputed(() => this.conversation.participating_user_ets());
 
-    this.should_render_cell = ko.pureComputed(() => {
-      return this.entered_viewport() && conversation.last_event_timestamp();
-    });
-
-    this.cell_state = ko.pureComputed(() => {
-      return this.should_render_cell() ? z.conversation.ConversationCellState.generate(this.conversation) : '';
-    }).extend({rateLimit: 100});
+    this.cell_state = ko.observable('');
+    this.cell_state_observable = ko.computed(() => {
+      window.requestAnimationFrame(() => this.cell_state(z.conversation.ConversationCellState.generate(this.conversation)));
+    }).extend({rateLimit: 500});
```

Comment:
coming soon to your nearest WebApp https://github.com/wireapp/wire-admin/blob/master/.eslintrc.yaml#L40 :)